CODEOWNERS

```diff
@@ -204,10 +204,10 @@
 /specification/scheduler/ @pinwang81
 
 # PRLabel: %Search
-/specification/search/data-plane/ @arv100kri @bleroy @AlexGhiondea @Mohit-Chakraborty @Azure/api-stewardship-board
+/specification/search/data-plane/ @arv100kri @bleroy @AlexGhiondea @Mohit-Chakraborty @Azure/api-stewardship-board @BevLoh
 
 # PRLabel: %Search
-/specification/search/resource-manager/ @abhi1509 @tjacobhi
+/specification/search/resource-manager/ @abhi1509 @tjacobhi @conor-joplin @BevLoh
```

Comment:
You should probably remove "abhi1509" from the code owners. They no longer work here.

Reply:
Thanks, updated